components/setting_picture.jsx

```diff
@@ -53,7 +54,7 @@ export default class SettingPicture extends Component {
 
     focusFirstElement() {
         if (this.settingList.current) {
-            const focusableElements = this.settingList.current.querySelectorAll('button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])');
+            const focusableElements = this.settingList.current.querySelectorAll('button, [href], input:not([class="hidden"]), select, textarea, [tabindex]:not([tabindex="-1"])');
```

Comment:
I'm thinking that `:not([class="hidden"])` might be too specific? If there are any other classes on that input (i.e. `class="hidden another-class"`) for whatever reason, the input will be matched even if it does have the `hidden` class. Though maybe that's what you want, in which case please ignore! 🙂

```suggestion
            const focusableElements = this.settingList.current.querySelectorAll('button, [href], input:not(.hidden), select, textarea, [tabindex]:not([tabindex="-1"])');
```

Reply:
So it's actually supposed to be `[tabindex]:not([tabindex="-1"])` that should be handling this case. I wanted to make sure I could stick `tabIndex` on that invisible input field to stop it not only being tabbed to, but also being auto focused. The problem was it just wouldn't work. Is there something I'm doing wrong here?